### ISSUE TITLE
No Support for 2Gbps/5Gbps in Direct Connect Hosted Connections

### ISSUE THREAD

**Issue by eddieramirez** [terraform-providers/terraform-provider-aws#12558]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
Terraform v0.12.21
+ provider.aws v2.54.0
```

### Affected Resource(s)

* aws_dx_connection
* aws_dx_lag

### Terraform Configuration Files

```hcl
resource "aws_dx_connection" "hoge" {
  name      = "tf-dx-connection"
  bandwidth = "5Gbps"
  location  = "EqDC2"
}
```

### Debug Output

```
Error: expected bandwidth to be one of [1Gbps 10Gbps 50Mbps 100Mbps 200Mbps 300Mbps 400Mbps 500Mbps], got 5Gbps
````

### Expected Behavior

* No error.

### Actual Behavior

* Error.

### Steps to Reproduce

1. `terraform apply`

### References

* https://docs.aws.amazon.com/directconnect/latest/UserGuide/WorkingWithConnections.html

**Comment by bflad** [comment]
Can confirm these values in this API documentation: https://docs.aws.amazon.com/directconnect/latest/APIReference/API_AllocateHostedConnection.html

If someone wants to update the Terraform validation, the code lives here: https://github.com/terraform-providers/terraform-provider-aws/blob/1d719e7407378a193769f2179198b85524444def/aws/validators.go#L1958-L1968

And ensure that any documentation in `website/docs/r/dx_connection.html.markdown` and `website/docs/r/dx_lag.html.markdown` is updated. Testing can be skipped for these particular resources.

**Comment by eddieramirez** [comment]
@bflad  I'm about to make a pull request. 👍

**Comment by bflad** [comment]
Support for the `2Gbps` and `5Gbps` values has been merged and will release with version 2.56.0 of the Terraform AWS Provider, later this week. Thanks to @eddieramirez for the implementation. 👍
